src/components/designer/custom/NodeInner.tsx

```diff
@@ -6,10 +6,12 @@ import { ThemeColors } from 'theme/colors';
 import { LOADING_NODE_ID } from 'utils/constants';
 import { Loader, StatusBadge } from 'components/common';
 import NodeContextMenu from '../NodeContextMenu';
+import { useStoreState } from 'store';
 
 const Styled = {
-  Node: styled.div<{ size?: ISize; colors: ThemeColors['node'] }>`
-    border: 1px solid ${({ colors }) => colors.border};
+  Node: styled.div<{ size?: ISize; colors: ThemeColors['node']; isSelected: boolean }>`
+    ${props =>
+      props.isSelected ? 'box-shadow: 2px 2px 5px rgb(255 147 40 / 40%)' : null};
```

Comment:
While testing, I started tweaking the box shadow a bit to see if I could improve it at all. I like the look more with the value `box-shadow: 5px 5px 20px rgb(255 147 40 / 40%)`. What do you think?

Reply:
Works for me.